src/components/inputs/localized-text-input/localized-text-input.js

```diff
@@ -116,173 +116,174 @@ const RequiredValueErrorMessage = () => (
 
 RequiredValueErrorMessage.displayName = 'RequiredValueErrorMessage';
 
-export default class LocalizedTextInput extends React.Component {
-  static displayName = 'LocalizedTextInput';
+const LocalizedTextInput = props => {
+  let defaultExpansionState = false;
 
-  static RequiredValueErrorMessage = RequiredValueErrorMessage;
+  if (props.hideLanguageExpansionControls) {
+    defaultExpansionState = true;
+  } else if (props.defaultExpandLanguages) {
+    defaultExpansionState = true;
+  }
```

Comment:
For me it would be easier to follow when inlining this at L129 and OR joining both prop values?

Reply:
Yeah it can be as simple as:

```js
const defaultExpansionState = props.hideLanguageExpansionControls || props.defaultExpandLanguages;
```